src/gwt/tools/sync-ace-commits

```diff
@@ -71,7 +71,6 @@ git merge \
     origin/patch/remove-webfont-2 \
     origin/patch/other-modes \
     origin/patch/dont-bundle-textmate-theme \
-    origin/bugfix/vim-ex-command-spellcheck \
```

Comment:
Why is this being removed?

Reply:
Evidently, that patch was introducing the duplicated setting of the `"spellcheck"` attribute to false. (I'm still not sure why this suddenly became an issue now -- perhaps one of the other patches somehow is sneaking that in? -- but since this fixes the issue and we're moving away from this mechanism for building Ace in v1.3 it should be fine)